coachmark/src/main/java/com/pseudoankit/coachmark/overlay/OverlayLayout.kt

```diff
@@ -0,0 +1,147 @@
+package com.pseudoankit.coachmark.overlay
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.UiComposable
+import androidx.compose.ui.layout.Layout
+import androidx.compose.ui.layout.Measurable
+import androidx.compose.ui.layout.Placeable
+import androidx.compose.ui.layout.layoutId
+import androidx.compose.ui.unit.Constraints
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.pseudoankit.coachmark.model.ToolTipPlacement
+import com.pseudoankit.coachmark.model.TooltipConfig
+
+/** Containing composable must use these values for layoutId on current and previous tooltip. */
+public enum class OverlayChildLayoutId { CURRENT, PREVIOUS }
```

Comment:
is this needed outside, can we make all internal?

Reply:
BTW isn't this layoutId for Tooltip, can we rename `TooltipId`
also do we need this as enum looks this can be a object and values as const
don't see any usage of having enums here